src/openvic-simulation/map/MapDefinition.cpp

```diff
@@ -198,20 +205,26 @@ bool MapDefinition::add_special_adjacency(
 	case LAND:
 	case STRAIT:
 		if (from.is_water() || to.is_water()) {
-			Logger::error(adjacency_t::get_type_name(type), " adjacency from ", from, " to ", to, " has water endpoint(s)!");
+			spdlog::error_s(
+				"{} adjacency from {} to {}",
```

Comment:
```suggestion
				"{} adjacency from {} to {} has water endpoint(s)!",
```
Missing part of the original message